### PR TITLE
fix(nix): update ALL THE QUERIES

### PR DESCRIPTION
**Pull request by clason**
let's try that again, shall we?

### CODE REVIEW

**Comment by clason** [comment]
Ok, I give up -- there are two highlight queries that still fail, and I can't see why from the changes to the `grammar.js`:

* `(select_expression ((variable_identifier) @_i (#eq? @_i "builtins")) (attrpath (identifier) @variable.builtin)) @variable.builtin`

* `(binary_expression "?" @operator)`

@theHamsta ?

**Comment by theHamsta** [comment]
I think `"?"` is now used in `has_attr_expression`, but i think there is no point of listing the kind of expression surrounding those operators and we could just always highlight them `[ "+" .... "?" ] @operator`

```
; unary operators
(unary_expression "-" @operator)
(unary_expression "!" @operator)

; binary operators
(binary_expression "++" @operator)
(binary_expression "*" @operator)
(binary_expression "/" @operator)
(binary_expression "+" @operator)
(binary_expression "-" @operator)
(binary_expression "//" @operator)
(binary_expression "<" @operator)
(binary_expression "<=" @operator)
(binary_expression ">" @operator)
(binary_expression ">=" @operator)
(binary_expression "==" @operator)
(binary_expression "!=" @operator)
(binary_expression "&&" @operator)
(binary_expression "||" @operator)
(binary_expression "->" @operator)

(has_attr_expression "?" @operator)
```

`(select_expression ((identifier) @_i (#eq? @_i "builtins"))` is missing an intermediate node of probably one of these
https://github.com/cstrahan/tree-sitter-nix/blob/470b15a60520ff7b86f51732b8d8f1118c86041e/grammar.js#L217-L218
I have no clue of nix. @leo60228 can you fix it?

**Comment by clason** [comment]
It was `variable_expression` :)

**Comment by clason** [comment]
And now the Windows CI runner complains that the nix parser isn't installed...

File: queries/nix/highlights.scm [hunk 1]
```diff
@@ -82,39 +82,40 @@
...
+(binary_expression "//" @operator)
+(binary_expression "<" @operator)
+(binary_expression "<=" @operator)
+(binary_expression ">" @operator)
+(binary_expression ">=" @operator)
+(binary_expression "==" @operator)
+(binary_expression "!=" @operator)
+(binary_expression "&&" @operator)
+(binary_expression "||" @operator)
+(binary_expression "->" @operator)
```
Comment (theHamsta) [inline queries/nix/highlights.scm:107]: Not change it to just the plain literals?

Comment (clason) [reply queries/nix/highlights.scm:107]: Since I can't test, I prefer not to make any changes beyond those absolutely necessary to unblock the updates.

**Comment by cstrahan** [comment]
As the author/maintainer of `tree-sitter-nix`, could I revive this? As I described in https://github.com/cstrahan/tree-sitter-nix/issues/27, the changes in `tree-sitter-nix` add support for a (relatively) recent addition to the official Nix syntax: interpolation/anti-quotations in paths. It would be nice for everyone to benefit from the new feature, so time permitting, I'd like to update the queries here so we can pull in the latest revision.

**Comment by theHamsta** [comment]
@cstrahan feel free to create a PR that makes the CI happy. Then, nix is back in

**Comment by leo60228** [comment]
I tried and failed to get something working myself. As the original author of Nix support, I'm happy to review a PR (though I don't have merge permissions or anything).

**Comment by theHamsta** [comment]
@clason should we merge this with Windows failing? Maybe ignore the check similar to swift?

**Pull request closed without merging**